### ISSUE TITLE
[request] option to disable "override changes" button

### ISSUE THREAD

**Issue by kmantel** [syncthing/syncthing-android#1882]
I use a few send-only folders and never want to use the "override changes" feature. While I understand there is now a confirmation dialog, I'd rather not even get that close with an accidental touch. A smaller added benefit would be uniformity of the folders interface.

**Comment by hugochinchilla** [comment]
Yeah, I just added a comment requesting the same in https://github.com/syncthing/syncthing-android/issues/1769#issuecomment-1708955380

> I use this to send all pictures from my android camera to my server and desktop PC. I set up the phone as send only because the server and desktop contain the pictures from previous phones, or content from the current phone before a periodic cleanup.
>
>In that context I would like to be able to disable or hide this action from the folder settings, so I cannot accidentally click on it. Having no confirmation dialog and being the biggest message on the screen is a very dangerous design IMHO.
>
>I have proper backups, but still... I don't want to need to restore from them (again).

**Comment by kmantel** [comment]
> Yeah, I just added a comment requesting the same in [#1769 (comment)](https://github.com/syncthing/syncthing-android/issues/1769#issuecomment-1708955380)
> 
> > I use this to send all pictures from my android camera to my server and desktop PC. I set up the phone as send only because the server and desktop contain the pictures from previous phones, or content from the current phone before a periodic cleanup.
> > In that context I would like to be able to disable or hide this action from the folder settings, so I cannot accidentally click on it. Having no confirmation dialog and being the biggest message on the screen is a very dangerous design IMHO.
> > I have proper backups, but still... I don't want to need to restore from them (again).

This is exactly one of the use cases I'd like to be able to disable override for

**Comment by cheintz** [comment]
> I understand there is now a confirmation dialog,

I think that's only in [the fork.](https://github.com/Catfriend1/syncthing-android)

I just lost data to the lack of a confirmation dialog on the "override changes" button today. Not only did I believe it would use changes to my local copy, I expected that anything with such an effect would have a confirmation dialog that would clearly explain what would happen. Yes, it's in the docs, but I hadn't edited my setup in months.
